meshapp/src/home/login.tsx

```diff
@@ -52,6 +53,13 @@ const SignUp = () => {
 };
 
 const Login = () => {
+
+  const [showForgotPassword, setShowForgotPassword] = useState(false);
+
+  const updateShowForgotPasswordState = () => {
+    setShowForgotPassword(prevState => !prevState)
```

Comment:
Wasn't this based on `showForgotPassword` previously? Should it be based on `prevState`?

Reply:
Oh nevermind I think I'm looking at the wrong section 😅

Reply:
https://stackoverflow.com/questions/73993973/understanding-prevstate-in-react